src/Constants/CountryCode.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace PodPoint\Countries\Constants;
```

Comment:
Not sure about a `Constants` namespace, I would just make an object called `CountryCodes` in the main namespace